requirements.txt

```diff
@@ -1,2 +1,5 @@
 Django==3.2.6
 uwsgi==2.0.19.1
+openpyxl
```

Comment:
Could you lock down the versions pls

Reply:
Alright fine I locked in the versions, pls take another look when you have a chance